### ISSUE TITLE
where to get partition.json this file?

### ISSUE THREAD

**Issue by Quillan11** [deyingk/agmn#1]
Hello! Thank you for this great work, and I am very interested in the code.  So I want to ask where can I get the file " ../data/external/cmu_panoptic_hands/intermediate_1/partition.json"? And it would be great that if you can provide more detailed training steps.

**Comment by deyingk** [comment]
Thank you for your interest! Sorry this repo is not well maintained. You can train the model following the steps below:

1. run train_unary_branch.py
2. run train_pairwise_branch.py
3. run train_agmn.py

After downloading the raw dataset from CMU Panoptic website, you might need to prepare the dataset using the following two python scripts under https://github.com/deyingk/agmn/tree/master/code/dataset/NotInFinal

a) crop.py
b) dataset_partition.py

Basically, they are just cropping the hand out of the original image (a 2.2x bounding box) and then partitioning them into training/val/test subsets.

**Comment by Quillan11** [comment]
Thanks a lot and what a great work!
